dev-perl/Gtk3-ImageView/Gtk3-ImageView-9.0.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DIST_AUTHOR=ASOKOLOV
+DIST_VERSION=9
+DIST_TEST=do
```

Comment:
Not needed unless you want to explicitly specify that the tests should run in sequence (i.e. not parallelized).
The default is DIST_TEST="do parallel"

Reply:
Yes, it's broken when in parallel, so I'm disabling parallelism